packages/standard-generator/src/fs.config.js

```diff
@@ -55,16 +55,6 @@ module.exports = {
         cacheDir: '.tmp/.pull-docs-last-page-plugin-cache'
       }
     },
-    {
-      modulePath: '@jpmorganchase/mosaic-plugins/NextPrevPagePlugin',
-      options: {
```

Comment:
I don't think these options have been ported to the Sidebar plugin.  Would be nice to keep them in some form...also add an option for `rootLevel`